applications/FluidDynamicsApplication/custom_strategies/schemes/bdf2_turbulent_scheme.h

```diff
@@ -270,7 +270,7 @@ class BDF2TurbulentScheme : public Scheme<TSparseSpace, TDenseSpace>
      * @param Dx Newton-Raphson iteration solution
      * @param b Newton-Raphson right hand side (unused)
      */
-    void Update(
+    virtual void Update(
```

Comment:
Why is the `virtual` required?

Reply:
Because this function is derived in bdf2_turbulent_schemeDEMCoupled.h, which will be merge to master after this PR.

Reply:
But this is already `virtual` in the base class... I mean, it makes no sense to add it also here.